style.css

```diff
@@ -515,3 +528,298 @@ input[type="text"] {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
 }
+
+@media screen and (min-width: 768px) {
+  .main-header {
+    display: flex;
+    justify-content: right;
+    margin-top: 30px;
+    flex-wrap: wrap;
+    margin-right: 155px;
+  }
+
+  .desk {
+    display: inline-block;
+    text-decoration: none;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 700;
+    font-size: 16px;
+    margin-right: 20px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .nav-container {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .ruby {
+    display: inline-block;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    background: #ebebff;
+    border-radius: 8px;
+    color: #6070ff;
+    font-weight: 222px;
+    font-size: 12px;
+    background-color: #ebebff;
+    font-family: 'Poppins', sans-serif;
+    padding: 8px 12px;
+    margin-right: 10px;
+  }
+
+  .part1 {
+    background: url(images/Header\ bg@2x.png);
+    background-position: cover;
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    margin-top: 50px;
+  }
+
+  .head-name {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin-left: 260px;
+    margin-top: 300px;
+  }
+
+  .head-para {
+    display: flex;
+    width: 920px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    margin-left: 260px;
+  }
+
+  .sub-head {
+    margin-left: 260px;
+  }
+
+  .social-media {
+    margin-left: 230px;
+  }
+
+  .logo {
+    margin-left: 155px;
+  }
+
+  .card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    justify-content: center;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    width: 90%;
+  }
+
+  .card img {
+    width: 95%;
+    height: 95%;
+    left: 24px;
+    top: 24px;
+  }
+
+  .title {
+    width: 496px;
+    height: 46px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin-left: 46px;
+    margin-top: 8px;
+    margin-right: 46px;
+  }
+
+  .field {
+    width: 297px;
+    height: 40px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 12px;
+    margin-left: 42px;
+  }
+
+  .about-myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .form {
+    height: 846px;
+    top: 5125px;
+    background: #6070ff;
+  }
```

Comment:
Your design
![image](https://user-images.githubusercontent.com/78251927/229274471-35c7248f-70cb-40b8-adfa-eb24539c5125.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/229274379-fddd3a7d-3b0d-4d18-b410-d3ef96826c05.png)

Please let's add the `background-image` of this section to match the template. 💪